.tours/3---dynamic flows.tour

```diff
@@ -0,0 +1,95 @@
+{
+    "title": "3 - Dynamic Flows",
+    "steps": [
+        {
+            "file": "es-space-mgmt/main/default/aura/reservationHelperAura/reservationHelperAura.cmp",
+            "description": "The `reservationHelper` Aura component render flow interviews dynamically, by using the `lightning:flow` base component. You can read more about `lightning:flow` [here](https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/components_using_flow_cmp.htm). We use Aura component embedding LWC component because LWC components do not provide ability to launch flows dynamically yet",
+            "line": 17,
+            "selection": {
+                "start": {
+                    "line": 17,
+                    "character": 1
+                },
+                "end": {
+                    "line": 20,
+                    "character": 1
+                }
+            }
+        },
+        {
+            "file": "es-space-mgmt/main/default/aura/reservationHelperAura/reservationHelperAuraHelper.js",
+            "description": "Use the `startFlow` method to start the create reservation flow with input variables",
+            "line": 14
+        },
+        {
+            "file": "es-space-mgmt/main/default/aura/reservationHelperAura/reservationHelperAura.cmp",
+            "description": "The `reservationHelper` Aura component embeds LWC component `reservationHelper`.",
```

Comment:
`reservationHelperAura` Aura component.

Reply:
May be add the next step here itself... It embeds this component and it acts as a bridge between.....

Reply:
Fixed the Aura Component. On the next one, It is there so it helps navigation to the LWC file so one can view the code. Merging them there is a chance to miss it.